src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -1357,7 +1377,7 @@ internal static Attribute GetAttribute(AttributeAst attributeAst)
             if (attributeType == null)
             {
                 throw InterpreterError.NewInterpreterException(attributeAst, typeof(RuntimeException), attributeAst.Extent,
-                    "CustomAttributeTypeNotFound", ParserStrings.CustomAttributeTypeNotFound, attributeAst.TypeName.FullName);
+                    "TypeNotFound", ParserStrings.TypeNotFound, attributeAst.TypeName.FullName);
```

Comment:
It is @lzybkr request https://github.com/PowerShell/PowerShell/pull/3784#discussion_r124377253